release-notes/9.0/preview/preview2/api-diff/Microsoft.NETCore.App/9.0-preview2_System.Threading.md

```diff
@@ -0,0 +1,17 @@
+# System.Threading
+
+``` diff
+ namespace System.Threading {
+     public static class Interlocked {
++        public static byte CompareExchange(ref byte location1, byte value, byte comparand);
++        public static short CompareExchange(ref short location1, short value, short comparand);
++        public static sbyte CompareExchange(ref sbyte location1, sbyte value, sbyte comparand);
++        public static ushort CompareExchange(ref ushort location1, ushort value, ushort comparand);
++        public static byte Exchange(ref byte location1, byte value);
++        public static short Exchange(ref short location1, short value);
++        public static sbyte Exchange(ref sbyte location1, sbyte value);
++        public static ushort Exchange(ref ushort location1, ushort value);
```

Comment:
The unsigned overloads have an `[System.CLSCompliantAttribute(false)]` attribute on them, should that also be shown here? LGTM otherwise.

Reply:
If the attribute was freshly added or removed, it should've showed up in the diff. Was that the case?

These are the only attributes I'm explicitly excluding (by request from past api diff PRs):
https://github.com/carlossanlop/arcade/blob/AsmDiffUpdates/src/Microsoft.DotNet.AsmDiff/AttributesToExclude.txt

I opened this issue to track the request: https://github.com/dotnet/arcade/issues/14585

Reply:
It looks like the APIs were added along with the attribute.